reference/shaders-hlsl/vert/basic.vert

```diff
@@ -0,0 +1,47 @@
+struct _UBO
+{
+    float4x4 uMVP;
+};
+
+cbuffer UBO
+{
+    _UBO _16;
+};
+uniform float4 gl_HalfPixel;
+
+static float4 gl_Position;
+static float4 aVertex;
+static float3 vNormal;
+static float3 aNormal;
+
+struct InputVert
+{
+    float3 aNormal : TEXCOORD0;
+    float4 aVertex : TEXCOORD1;
+};
+
+struct OutputVert
+{
+    float3 vNormal : TEXCOORD0;
+    float4 gl_Position : POSITION;
+};
+
+void vert_main()
+{
+    gl_Position = mul(transpose(_16.uMVP), aVertex);
```

Comment:
According to https://msdn.microsoft.com/de-de/library/windows/desktop/bb509628(v=vs.85).aspx this should actually work. Will have to do some tests and think hard about what else could break.